modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json

```diff
@@ -69,6 +69,1310 @@
     }
   },
   "paths": {
+    "/power-of-attorney-requests/{id}/decision": {
+      "post": {
+        "summary": "Create the decision for Power of Attorney requests.",
+        "tags": [
+          "Power of Attorney"
+        ],
+        "operationId": "createPowerOfAttorneyRequestDecisions",
+        "security": [
+          {
+            "productionOauth": [
+              "system/claim.read",
+              "system/system/claim.write"
```

Comment:
typo? it's in multiple spots as well - I know I saw some of this in another section of the documentation so assuming it was carried over.